src/puppeteer_pool.js

```diff
@@ -15,6 +15,7 @@ const PROCESS_KILL_TIMEOUT_MILLIS = 5000;
 const PAGE_CLOSE_KILL_TIMEOUT_MILLIS = 1000;
 
 const DEFAULT_OPTIONS = {
+    reusePages: true,
```

Comment:
Is it safe to enable this by default in the very first version? I think it's better to have this flag there for some time, tell people to try it for their use cases and only then enable it by default.

Reply:
That's a fair point. I wanted to release a new minor (effectively major) version with the change, along with all the other PRs that are in the pipeline, but we can turn it off too for starters.